mavros/src/plugins/waypoint.cpp

```diff
@@ -541,6 +545,7 @@ class WaypointPlugin : public plugin::PluginBase {
 		}
 	}
 
+	// @brief Act on first heartbeat from FCU
 	void connection_cb(bool connected) override
```

Comment:
Override functions do not need special documentation (exception if it do some not obvious things).

Reply:
@JMare just like this:
`// Act on first heartbeat from FCU` (no `@brief` and no `//!`, which results in it not being processed on Doxygen)